clusterloader2/pkg/measurement/common/network/network_performance_measurement.go

```diff
@@ -396,3 +399,20 @@ func (npm *networkPerformanceMeasurement) getMetricData(data *testResultSummary,
 	data.DataItems = append(data.DataItems, metricDataItem)
 	klog.V(3).Infof("TestResultSummary: %v", data)
 }
+
+func (npm *networkPerformanceMeasurement) waitForMetricsFromPods() {
+	bufferTime := time.Duration(math.Max(10, float64(npm.numberOfClients)*0.1)) * time.Second
+	timeout := initialDelayForTestExecution + npm.testDuration + bufferTime
+	interval := 2 * time.Second
+	if err := wait.Poll(interval, timeout, npm.checkResponseReceivedFromPods); err != nil {
+		// TODO: Log pod names from which response is not received.
+		klog.Errorf("Failed to receive response from all pods: %v", err)
```

Comment:
Given the check function never returns the error - the err will always be the timeout.

So it's enough to have:

klog.Errorf("Failed to receive response from %v pods", <value>)

Reply:
done